CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Bugs fixed
 - Ensure that Web3Signer stops the http server when a sigterm is received
+- Improve Key Manager API import operation to use parallel processing instead of serial processing.
```

Comment:
think this is more of an improvement than a bug but you might disagree :)

Reply:
will move it to Improvements section